docs/install-run/install.rst

```diff
@@ -0,0 +1,316 @@
+
+==================================
+Installation
+==================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+=============
+
+CrateDB requires a `Java virtual machine`_ to run.
+
+- For CrateDB version 4.2 and newer
+   
+    - `Java virtual machine`_ is bundled with CrateDB and no extra installation is necessary.
+
+- For CrateDB version 3.0 to 4.1 
+   
+   - `Java 11`_ installation is required. We recommend using `Oracle's Java`_ on macOS and Microsoft Windows and OpenJDK_ on Linux Systems. Earlier versions required Java 8.
+
+
+
+Basic Installation
+===================
+
+For Linux, Micrsoft Windows and macOS
+
+#. Download `the latest CrateDB release`_.
```

Comment:
Do you think it would make sense to add an extra remark about picking the right Tarball depending on the system? Based on some of the issues we see it's not that uncommon that people download the wrong one.

Reply:
:+1:  I think it never really got into the docs, that there are now different tarballs/zips for different OSs :)
We should definitly add a remark and then either link to the CDN or the general download page (https://crate.io/download/)